app/components/view-epics/macro.njk

```diff
@@ -0,0 +1,99 @@
+{% from 'components/tag/macro.njk' import tag %}
+{% from 'components/view-data-title/macro.njk' import viewDataTitle %}
+{% from 'components/view-epic-list/macro.njk' import viewEpicList %}
+
+{% macro viewEpics(params) %}
+  {% if params.data %}
+    <div data-test-id="{{ params.dataTestId }}" class="bc-c-epics {{ params.classes }}">
+      <div class="nhsuk-grid-row">
+        {% if params.data.must %}
+          <div class="must-epics nhsuk-grid-column-one-half">
+            <div>
+              {{ tag({
+                dataTestId: "example",
```

Comment:
the data-test-id needs changed?

Reply:
yes this should be changed. `must-tag` perhaps?